plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
```

Comment:
I did that on purpose to reflect the values that are written in the ini-File. This is important if you insert new elements which would break ini-compatibility. But I am open to remove these manual assignments. Maybe here a third opinion ( @FlatOutPS2 @turtleli )?